src/pages/index.js

```diff
@@ -94,8 +162,53 @@ addCardButton.addEventListener("click", () => {
 });
 
 editProfileButton.addEventListener("click", () => {
+  api.loadUserInfo().then(({ name, about }) => {
```

Comment:
No need to call `api.loadUserInfo()` when you open the profile modal. You have method `getUserInfo` for that not to waste resources